tests/classic/task.yaml

```diff
@@ -0,0 +1,41 @@
+summary: Run ubuntu-image classic command tests
+execute: |
+    IMG_DEF_PATH=/home/ubuntu-image/ubuntu-images/ubuntu-server-pc-amd64.yaml
+
+    ubuntu-image classic --debug --workdir /tmp/workdir $IMG_DEF_PATH
+
+    IMG_NAME=$(cat $IMG_DEF_PATH | yq '.artifacts.img.[0].name')
+    mkdir /tmp/loopback
+    losetup --find --show --partscan /tmp/workdir/$IMG_NAME
+    losetup -a |grep $IMG_NAME|cut -f1 -d: >> loop.txt
+    LOOP="$(cat loop.txt)"
+
+    for l in `ls -d "$LOOP"p*`
+    do 
+        p=${l#"$LOOP"}
+        mkdir /tmp/$p
+        mount $l /tmp/$p || true
+    done
+
+    ls /tmp/p3/
+
```

Comment:
I recommend to add a debug section to see some information that could help to determine why the tests failed. For example the disk space, mounts, the journal log, etc

Reply:
Good idea. Are there any tools to do that in a nice way? Do you have an example somewhere?